src/main/java/practice/CandidateValidator.java

```diff
@@ -1,5 +1,20 @@
 package practice;
 
-public class CandidateValidator {
-    //write your code here
+import java.util.function.Predicate;
+import model.Candidate;
+
+public class CandidateValidator implements Predicate<Candidate> {
+    private static final int PERIOD_TO_NUMBER = 1;
+    private static final int PERIOD_FROM_NUMBER = 0;
+    private static final int PERIOD_IN_UKRAINE = 10;
+
+    @Override
+    public boolean test(Candidate c) {
```

Comment:
```suggestion
    public boolean test(Candidate candidate) {
```
always go for readability - we, as devs, more often read code, than write it